### PR TITLE
Add community to Hugo

### PR DESCRIPTION
**Pull request by maxammann**
I added now the community page directly to this repo. During building all profile pictures are fetched and processed.
The page  loads now super fast.

### CODE REVIEW

**Comment by wipfli** [comment]
Can you merge main?

File: assets/main.js [hunk 1]
```diff
@@ -1,4 +1,39 @@
...
+    style: "https://demotiles.maplibre.org/style.json",
+    center: [0, 0],
+    zoom: 1,
+    hash: true,
+    attributionControl: false,
+  }).addControl(
+    new AttributionControl({
+      compact: false,
+      customAttribution:
+        'OpenStreetMap contributors. | <a href="https://github.com/maplibre/community">Edit on GitHub.</a>',
```
Comment (wipfli) [inline assets/main.js:20]:
```suggestion
        'OpenStreetMap contributors. | <a href="https://github.com/maplibre/maplibre.github.io">Edit on GitHub.</a>',
```

**Review by wipfli — CHANGES_REQUESTED** [review]
Thanks for migrating this to the hugo website. Please fix the url as suggested. I could not build locally:

```
➜  maplibre.github.io git:(contributors) ✗ npm run serve

> maplibre@1.0.0 serve
> hugo serve

Start building sites … 
hugo v0.98.0-165d299cde259c8b801abadc6d3405a229e449f6 linux/amd64 BuildDate=2022-04-28T10:23:30Z VendorInfo=gohugoio
WARN 2022/05/07 13:54:13 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/05/07 13:54:13 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
Error: Error building site: TOCSS: failed to transform "style.scss" (text/x-scss). Check your Hugo installation; you need the extended version to build SCSS/SASS.: this feature is not available in your current Hugo version, see https://goo.gl/YMrWcn for more information
Built in 17121 ms
```

**Comment by wipfli** [comment]
Ah I just had to re-run `./hugow --get-extended`

**Comment by wipfli** [comment]
Can you merge main? This version has separate news and blog again...
image

**Comment by maxammann** [comment]
The unified Blog and News was added in #25

**Comment by wipfli** [comment]
Can you resolve the conflict? This looks good to me otherwise.

**Comment by maxammann** [comment]
Should be fine now

**Review by wipfli — APPROVED** [review]
Thanks!

**Comment by maxammann** [comment]
@wipfli it seems like the community page was overwritten again by the other repository. Can you take a look?

**Comment by wipfli** [comment]
Oh good catch. I will turn pages of there...

**Comment by wipfli** [comment]
Should be fixed now.

**Comment by maxammann** [comment]
I did rerun the Github Action, works now, thanks!

**Pull request merged**